src/resonanceReconstruction/rmatrix/legacy/unresolved/ResonanceTable/src/verifyTable.hpp

```diff
@@ -0,0 +1,27 @@
+static
+void verifyTable( const std::vector< Resonance >& resonances,
+                  const Degrees& degrees ) {
+
+  if ( resonances.size() < 2 ) {
+
+    Log::error( "There must be at least two resonances in an unresolved "
+                "resonance table to interpolate on the parameters" );
+    Log::info( "Found {}", resonances.size() );
+    throw std::exception();
+  }
+
+  const auto verifyDegreesOfFreedom = [] ( const auto& entry ) {
+
+    if ( entry > 4 ) {
+
+      Log::error( "The degrees of freedom for each channel must between 0 and 4" );
```

Comment:
where do you check if the degrees of freedom are `<=0`?

Reply:
It's unsigned ...